video_xblock/tests/unit/mocks/wistia.py

```diff
@@ -38,208 +32,3 @@ def get(self):
         if self.event == 'not_authorized':
             self.return_value = ResponseStub(status_code=401)
         return lambda x: self.return_value
-
-
-class WistiaDefaultTranscriptsMock(BaseMock):
```

Comment:
Fair enough.
So you've replaced these tests with `WistiaPlayerTest` which cover the same code and cases, but is simpler, right?